Trust and compliance: Security section (#104)
pretix/pretix-docs: docs/trust/security/index.md

# Information security

> Security of your data is of the utmost importance to us both on a technical and organizational level.

Reviewer: ```suggestion
The security of your data is of the utmost importance to us—both on a technical and organizational level.
```

> On the pages in this section, we describe some of the measures we take in more detail and give information on how you can ensure compliance on your end as well.

Reviewer: ```suggestion
On the pages in this section, we describe the security measures we take in more detail
We also provide information on how you can ensure compliance on your end. 
```
compliance mit was?

Author: Mit was auch immer der Kunde so compliant sein muss, helfen ihm die Infos hoffentlich. Also, unsere ISO27001-Zertifizierung hilft ihm ja nicht dabei, sicher zu sein, sondern dabei, compliant zu sein, mit den Anforderungen seiner Stakeholder wiederum.
